themes-default/slim/views/inc_defs.mako

```diff
@@ -82,3 +82,19 @@
     else:
         cssClass = overrideClass
 %><span${title} class="${cssClass}">${qualityString}</span></%def>
+
+<%def name="convert(obj)">
+    <%
+    import json
+    ## This converts the keys to strings as keys can't be ints
+    print('test')
```

Comment:
Remove?

Reply:
Haha. I couldnt find that one anymore. Tnx